translations/en.json

```diff
@@ -335,6 +335,9 @@
   "change-password-old-password-placeholder": "Enter current password",
   "change-password-repeat-password-placeholder": "Repeat new password",
   "change-pin": "Change 6-digit passcode",
+  "change-pin-keycard": "Change PIN",
+  "change-pin-keycard-description": "Use different PIN for this Keycard",
+  "change-pin-keycard-message": "It’s very important that you will not forget new PIN. Status is unable to restore PIN. If you forget it, Keycard will be blocked and you will need to restore it via recovery phrase or PUK.",
```

Comment:
Although I'm not a native speaker and I make plenty of mistakes when I write, I see a few small English issues in this sentence. I prefer if we nail the text straight from the very first PR doing the implementation (also to avoid rework).

Personally, I immediately raise a red flag when I use apps with certain grammatical mistakes. I've seen other translations with less than ideal English text in the Keycard screens or which could feel more natural (separate problem).

We can further discuss this with @benjthayer outside this PR. Not a blocker for this PR @Parveshdhull, but something we should improve across all features we work on. cc @xAlisher